go.mod

```diff
@@ -274,7 +274,7 @@ require (
 	k8s.io/apiserver v0.26.2 // indirect
 	k8s.io/component-base v0.26.2 // indirect
 	k8s.io/gengo v0.0.0-20211129171323-c02415ce4185 // indirect
-	k8s.io/klog/v2 v2.120.1 // indirect
+	k8s.io/klog/v2 v2.130.1 // indirect
```

Comment:
Non blocker, but why did this need to be bumped?

Reply:
I needed to bring in openshift/api release-4.12 or newer to have access to the ControlPlaneMachineSet struct.

Reply:
Thanks for the clarification. Not that keeping dependencies current is at all bad, but I didn't see the exact cause immediately in my review so I was just curious.